tests/aequilibrae/transit/rebuild_transit_example.ipynb

```diff
@@ -0,0 +1,137 @@
+{
```

Comment:
Are you sure this notebook s part of the PR?

Reply:
I was asked to put this in the repo as an example - see AequilibraE Teams chat under General.